pep-9999.rst

```diff
@@ -149,6 +149,61 @@ Two kinds of structured comments are accepted:
 * A ``# type: ignore`` comment at the end of any line, which suppresses all type
   errors in that line.
 
+Version and Platform Checks
+---------------------------
+
+Type stubs for libraries that support multiple Python versions can use version
+checks to supply version-specific type hints. Type stubs for different Python
+versions should still conform to the most recent Python version's syntax, as
```

Comment:
"should still conform to the most recent *supported* Python version's syntax", maybe? since we decided to change the syntax section